content/en/apps/guides/data/csv-to-docs.md

```diff
@@ -186,6 +186,38 @@ The resulting doc would be as follows, with the `_id` from `district_1` as the `
   "_id": "45293356-353c-5eb1-9a41-baa3427b4f69"
 }
 ```
+##### Link to a parent document already existing in couchdb
+You may wish to link the new record to a parent document that already exists in couchdb that was created in the past.
```

Comment:
CouchDB is nowhere introduced or mentioned in this guide. Although it should not be a problem for more experienced readers, is there a possibility that it can create confusion for some readers? Should we consider rewording it to something else, such as `document that already exists in the app` or `the database`?

Reply:
It is optional, but did you miss this comment? @pkitutu